src/Catalyst.Core.Modules.Kvm/DeltaExecutor.cs

```diff
@@ -225,43 +205,21 @@ private void Execute(ContractEntry entry, StateUpdate stateUpdate, ITxTracer txT
             var gasLimit = entry.GasLimit;
             var intrinsicGas = CalculateIntrinsicGas(entry, spec);
 
-            if (_logger.IsEnabled(LogEventLevel.Verbose))
-            {
-                _logger.Verbose("Executing entry {entry}", entry);
-            }
+            if (_logger.IsEnabled(LogEventLevel.Verbose)) _logger.Verbose("Executing entry {entry}", entry);
 
-            if (!ValidateSender(entry, env, txTracer))
-            {
-                return;
-            }
+            if (!ValidateSender(entry, env, txTracer)) return;
 
-            if (!ValidateIntrinsicGas(entry, env, intrinsicGas, txTracer))
-            {
-                return;
-            }
+            if (!ValidateIntrinsicGas(entry, env, intrinsicGas, txTracer)) return;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4614986)